spec/std/llvm/aarch64_spec.cr

```diff
@@ -6,7 +6,7 @@ LLVM.init_aarch64
 {% end %}
 
 private def abi
-  triple = "aarch64-unknown-linux-gnu"
+  triple = LLVM.default_target_triple.gsub(/^(.+?)-/, "aarch64-")
```

Comment:
Not 100% on this change, why does it need to happen?

Reply:
I’m not sure about it too, but hard-coded `aarch64-unknown-linux-gnu` is not very useful for testing aarch64 with musl, isn’t it?

Reply:
(I’ve copied this from https://github.com/crystal-lang/crystal/blob/9adbb92236fddcf34799f656139a801817f2aad7/spec/std/llvm/x86_64_abi_spec.cr#L9).

Reply:
I think these specs should be changed to always test all the triples we support...

However, leaving it as-is is fine for now, since there's no difference between the calling convention and struct layout for musl and glibc.

Reply:
Okay, I’ve reverted this change.